ed_gui_server/src/gui_server_plugin.cpp

```diff
@@ -537,4 +562,146 @@ bool GUIServerPlugin::srvInteract(const ed_gui_server_msgs::Interact::Request& r
     }
 }
 
+// ----------------------------------------------------------------------------------------------------
+
+bool GUIServerPlugin::srvMap(const ed_gui_server_msgs::Map::Request& req,
+                             ed_gui_server_msgs::Map::Response& res)
+{
+    geo::Vec2 p_min(1e9, 1e9);
+    geo::Vec2 p_max(-1e9, -1e9);
+
+    bool model_found = false;
```

Comment:
What does this boolean value mean? I can see that if it is false the walls will be used to bound the image. Is that the desired behaviour? If so why does the user not simply specify the walls as requested entities in view?

Reply:
In case none of the requested models are found, just include the entire WM.